spec/factories/mission.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :mission, class: 'Mission' do
```

Comment:
Nit: 應該不需要指定 class 也可以。
```suggestion
  factory :mission do
```